modules/nodes-secondary-scheduler-configuring-console.adoc

```diff
@@ -0,0 +1,71 @@
+// Module included in the following assemblies:
+//
+// * nodes/scheduling/secondary_scheduler/nodes-secondary-scheduler-configuring.adoc
+
+:_content-type: PROCEDURE
+[id="nodes-secondary-scheduler-configuring-console_{context}"]
+= Deploying a secondary scheduler
+
+After you have installed the {secondary-scheduler-operator}, you can deploy a secondary scheduler.
+
+.Prerequisities
+
+* You have access to the cluster with `cluster-admin` privileges.
+* You have access to the {product-title} web console.
+* The {secondary-scheduler-operator-full} is installed.
+
+.Procedure
+
+. Log in to the {product-title} web console.
+. Create config map to hold the configuration for the secondary scheduler.
+.. Navigate to *Workloads* -> *ConfigMaps*.
+.. Click *Create ConfigMap*.
+.. In the YAML editor, enter the config map definition that contains the necessary `KubeSchedulerConfiguration` configuration. For example:
++
+[source,yaml]
+----
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: "secondary-scheduler-config"                  <1>
+  namespace: "openshift-secondary-scheduler-operator" <2>
+data:
+  "config.yaml": |
+    apiVersion: kubescheduler.config.k8s.io/v1beta3
+    kind: KubeSchedulerConfiguration                  <3>
+    leaderElection:
+      leaderElect: false
+    profiles:
+      - schedulerName: secondary-scheduler            <4>
+        plugins:                                      <5>
+          score:
+            disabled:
+              - name: NodeResourcesBalancedAllocation
+              - name: NodeResourcesLeastAllocated
+----
+<1> The name of the config map. This is used in the *Scheduler Config* field when creating the `SecondaryScheduler` CR.
+<2> The config map must be created in the `openshift-secondary-scheduler-operator` namespace.
+<3> The `KubeSchedulerConfiguration` resource for the secondary scheduler. For more information, see link:https://kubernetes.io/docs/reference/config-api/kube-scheduler-config.v1beta3/#kubescheduler-config-k8s-io-v1beta3-KubeSchedulerConfiguration[`KubeSchedulerConfiguration`] in the Kubernetes API documentation.
+<4> The name of the secondary scheduler. Pods that set their `spec.schedulerName` field to this value are scheduled with this secondary scheduler.
+<5> The plug-ins to enable or disable for the secondary scheduler. For a list default scheduling plug-ins, see link:https://kubernetes.io/docs/reference/scheduling/config/#scheduling-plugins[Scheduling plugins] in the Kubernetes documentation.
++
+// TODO: Also include link to https://github.com/kubernetes-sigs/scheduler-plugins, but need to know what this should be called
```

Comment:
@sferich888 It was requested that we link to this list of scheduler plug-ins on GitHub (for these docs on the new secondary scheduler operator. Are you okay with us adding this GitHub link (and if so, is there any special wording around it you'd like to see)? Thanks!

Reply:
I don't like the word `default` in this copy. This implies that we are shipping / delivering plugins (and because of that I would say; linking to github isn't appreciate. Things that we deliver to customers (as options) should be fully supported/documented (meaning that we should mirror the upstream docs downstream if we must). 

As I understand it we don't deliver these; thus these are just 'examples' or 'community plugins' that customers can use or explore.  Thus if we re-frame this to explain these are community unsupported examples; then linking to github is fine (but I am still not enthusiastic about it).